SPECS/lld/lld.spec

```diff
@@ -0,0 +1,72 @@
+Summary:        LLD is a linker from the LLVM project that is a drop-in replacement for system linkers and runs much faster than them
+Name:           lld
+Version:        12.0.1
+Release:        1%{?dist}
+License:        NCSA
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://clang.llvm.org
+Source0:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/lld-%{version}.src.tar.xz
+Source1:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/libunwind-%{version}.src.tar.xz
+BuildRequires:  build-essential
+BuildRequires:  cmake
+BuildRequires:  llvm-devel
+BuildRequires:  ninja-build
+BuildRequires:  python3
+BuildRequires:  file
+
+%package devel
+Summary:        Libraries and header files for LLD
+Requires:       %{name} = %{version}-%{release}
+
+%package libs
+Summary:        LLD shared libraries
+
+%description
+The LLVM project linker.
+
+%description devel
+This package contains library and header files needed to develop new native
+programs that use the LLD infrastructure.
+
+%description libs
+Shared libraries for LLD.
+
+%prep
+%setup -q -b 1 -n lld-%{version}.src
+
+%build
+mkdir -p build
+cd build
+%cmake ..                                                         \
+       -G Ninja                                                   \
+       -DCMAKE_BUILD_TYPE=Release                                 \
+       -DCMAKE_SKIP_RPATH:BOOL=on                                 \
+       -DCMAKE_C_FLAGS=-I../../libunwind-%{version}.src/include   \
+       -DCMAKE_CXX_FLAGS=-I../../libunwind-%{version}.src/include \
+       -DLLVM_LINK_LLVM_DYLIB:BOOL=on                             \
+       -DLLVM_DYLIB_COMPONENTS="all"                              \
+       -Wno-dev
+%ninja_build
+
+%install
+cd build
+%ninja_install
+
+%files
+%license LICENSE.TXT
+%{_bindir}
```

Comment:
If we don't want to enumerate the specific binaries here, can we use the `%{_bindir}/*` instead? The way it is currently, the `lld` package will have ownership over the `/usr/bin` folder.

Reply:
Oh, interesting, I wasn't aware of that behaviour, I'll change.